cpp/tests/test_odeseir.cpp

```diff
@@ -216,6 +177,14 @@ TEST(TestOdeSeir, get_reproduction_numbers)
     mio::TimeSeries<ScalarType>::Vector result_5(4);
     mio::TimeSeries<ScalarType>::Vector result_6(4);
 
+    result_0.setZero();
```

Comment:
Do we need this? Isn't zero the default value?

Reply:
This was marked by valgrind :)